lib/user_mappings.yml

```
@@ -26,3 +26,4 @@ tomoya55: W8W4GBHC5
 yuseinishiyama: W8VE975JM
 ollieh-m: WEH1S8VMW
 chrisbr: WFWHS2W1M
+teja-sophista: WHMG60P47
```

(is your GitHub handle not @tejanium ? 😅 )

```suggestion
tejanium: WHMG60P47
```

---

ah 😓 
thank you for this